### PR TITLE
YBRO 3 more

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/fallaji_antiquarian.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Fallaji Antiquarian
+ManaCost:3 R
+Types:Creature Human Artificer
+PT:2/4
+K:Haste
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | IsPresent$ Card.Self | NoResolvingCheck$ True | Execute$ TrigConjure | TriggerDescription$ When CARDNAME enters the battlefield, conjure a duplicate of another target nontoken creature or artifact you control into your graveyard. The duplicate perpetually gains unearth {1}{R}.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fallaji_antiquarian.txt:6]: Why IsPresent + NoResolvingCheck?

File: forge-gui/res/cardsfolder/upcoming/raddic_tal_zealot.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ManaCost:2 W B
+Types:Legendary Creature Human Knight
+PT:4/4
+K:Hexproof:Card.Black:white
+K:Hexproof:Card.Black:black
+T:Mode$ AttackersDeclared | ValidAttackers$ Knight.YouCtrl |Execute$ TrigDraft | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more Knights you control attack, draft a card from CARDNAME's spellbook.
+SVar:TrigDraft:DB$ Draft | Spellbook$ Midnight Reaper,Guardian of Faith,Knight of the Ebon Legion,Cavalier of Dawn,Benalish Marshal,Cavalier of Night,Murderous Rider,Order of Midnight,Acclaimed Contender,Dauntless Bodyguard,Valiant Knight,Smitten Swordmaster,Blacklance Paragon,History of Benalia,The Circle of Loyalty
+DeckHints:Type$Knight
+DeckHas:Type$Saga|Enchantment|Vampire|Artifact|Zombie|Elemental & Ability$Graveyard|LifeGain|Token|Counters
+Oracle:Hexproof from white, Hexproof from black.\nWhenever one or more Knights you control attack, draft a card from Raddic, Tal Zealot's spellbook.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/raddic_tal_zealot.txt:11]: capitalization for the hexproof looks wrong

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/raddic_tal_zealot.txt:11]: Oh, gross... Arena's way of doing this is different than paper *sigh* this is probably good enough

just get rid of the period after `black.`

File: forge-gui/res/cardsfolder/upcoming/rusko_clockmaker.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Name:Rusko Clockmaker
+ManaCost:2 U B
+Types:Legendary Creature Human Artificer
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | IsPresent$ Card.Self | NoResolvingCheck$ True | Execute$ TrigConjure | TriggerDescription$ When CARDNAME enters the battlefield, conjure a card named Midnight Clock onto the battlefield.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/rusko_clockmaker.txt:5]: same with IsPresent / NoResolvingCheck

**Review by Northmoc — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**